subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc

```diff
@@ -108,6 +108,21 @@ Previously Gradle used to detect if an input file collection annotated with `@Sk
 To ignore directories in Gradle 8.0 and later, the input property needs to be explicitly annotated with `@IgnoreEmptyDirectories`.
 For more information see <<empty_directories_file_tree>>.
 
+==== Project facets generated by the `eclipse wtp' plugin have changed
```

Comment:
```suggestion
==== Adding `jst.ejb` with the `eclipse wtp' plugin now removes the `jst.utility` facet
```
Suggestion: the proposed title is short, but I think it could be clearer. Perhaps a reword can provide additional detail without getting too lengthy.